crates/python/qcs_sdk/qpu/client.pyi

```diff
@@ -0,0 +1,65 @@
+from typing import Optional
+
+
+class QcsClient:
+    """
+    Configuration for connecting and authenticating to QCS API resources.
+    """
+
+    def __new__(
+        tokens: Optional[QcsClientTokens] = None,
+        api_url: Optional[str] = None,
+        auth_server: Optional[QcsClientAuthServer] = None,
+        grpc_api_url: Optional[str] = None,
+        quilc_url: Optional[str] = None,
+        qvm_url: Optional[str] = None,
+    ) -> "QcsClient":
+        """
+        Construct a client from scratch.
+        
+        Use ``QcsClient.load`` to construct an environment-based profile.
+        """
+        ...
+    
+    @staticmethod
+    def load(
+        profile_name: Optional[str] = None,
+        use_gateway: Optional[bool] = None,
+    ) -> "QcsClient":
+        """
+        Load a QcsClient configuration using an environment-based configuration.
+
+        See for details: https://docs.rigetti.com/qcs/references/qcs-client-configuration#environment-variables-and-configuration-files
+        """
+        ...
+
+
+class QcsClientAuthServer:
+    """Authentication server configuration for the QCS API."""
+
+    client_id: str
+    issuer: str
+
+
+
+class QcsClientTokens:
```

Comment:
Does this type hint line up with the implementation? It looks like the conversion to the Rust type is from `PyDict`. Maybe [TypedDict](https://docs.python.org/3/library/typing.html#typing.TypedDict) is a better fit?

Reply:
Same for `QcsClientAuthServer`

Reply:
Good catch, fixed!